example/src/Application/index.tsx

```diff
@@ -16,7 +15,29 @@ import {
 } from '@react-navigation/native';
 import { Theme } from '@react-navigation/native/lib/typescript/src/types';
 import ApplicationRouter from './ApplicationRouter';
-import { IS_IOS } from '../Data/Constants';
+import { useAsyncStorage } from '@react-native-async-storage/async-storage';
+import {
+  IS_IOS,
+  DEFAULT_SETTINGS_DATA,
+  STORAGE_SETTINGS_KEY,
+} from '../Data/Constants';
+import { Buffer } from 'buffer';
+import { LaunchArguments } from 'react-native-launch-arguments';
+interface MyExpectedArgs {
+  customSettings?: string;
+}
+const args = LaunchArguments.value<MyExpectedArgs>();
+
+const getSettingsFromEnv = () => {
```

Comment:
Does this section need more checks?

Reply:
try/catch block will help with catching exceptions when the JSON is malformed